tests/foreman/cli/test_docker.py

```diff
@@ -1633,15 +1633,16 @@ def test_positive_pull_image(self):
         repo = Repository.info({'id': repo['id']})
         try:
             # publishing takes few seconds sometimes
-            retries = 10 if bz_bug_is_open(1452149) else 1
-            for i in range(retries):
-                result = ssh.command(
-                    'docker pull {0}'.format(repo['published-at']),
-                    hostname=self.docker_host.ip_addr
-                )
-                if result.return_code == 0:
-                    break
-                sleep(2)
+            result, _ = wait_for(
+                    lambda: ssh.command(
+                            'docker pull {0}'.format(repo['published-at']),
+                            hostname=self.docker_host.ip_addr
+                    ),
+                    num_sec=20 if bz_bug_is_open(1452149) else 1,
```

Comment:
1452149 is in Closed state, so that condition make no sense

Reply:
Yes, but it's Closed Wontfix, meaning the problem that prompted us to add this condition has not been properly resolved. Thought this function is able to tell the difference.

I'll just set it unconditionally, then. Unless you know better solution?